crates/sui-types/src/accumulator_root.rs

```diff
@@ -53,8 +52,8 @@ pub fn get_accumulator_root_obj_initial_shared_version(
 /// Rust type for the Move type AccumulatorKey used to derive the dynamic field id for the
 /// balance account object.
 #[derive(Debug, Serialize, Deserialize, Clone)]
-struct AccumulatorKey {
-    owner: SuiAddress,
+pub struct AccumulatorKey {
```

Comment:
It's not super clear from the name on the difference between AccumulatorKey and OwnerKey. Might be worth adding Metadata prefix on the metadata structs, and maybe even move the metadata code to a separate file for clarity.

Reply:
moved to a separate file